dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -5,7 +5,6 @@
 FROM {{ .from }}
 
 RUN yum -y --setopt=tsflags=nodocs update && \
-    yum install -y epel-release jq \
     yum clean all
```

Comment:
```suggestion
    yum install -y epel-release \
    yum install -y jq \
    yum clean all
```

Reply:
this wont' work, see my comment below

Reply:
Your comment below doesn't give a reason why this doesn't work? I am saying the same thing you are doing below but move it up here after `yum clean all`.

It just needs to be changed to be split into to commands, they do nothing each need to be in its own layer, and no reason `yum clean all` cannot be called.